aggregator/src/aggregator.rs

```diff
@@ -180,6 +190,8 @@ pub struct Aggregator<C: Clock> {
     /// Counter tracking the number of failed message decodes while handling the
     /// `tasks/{task-id}/reports` endpoint.
     upload_decode_failure_counter: Counter<u64>,
+    /// Counter tracking the number of successfully-aggregated reports.
+    aggregation_success_counter: Counter<u64>,
```

Comment:
We might want to put all these `Counter<T>`s behind some kind of `struct AggregatorMetrics`, so that we'd have fewer functions to update in the future when we add new metrics. This would also give us more runway before we hit `too_many_arguments` lints.